less/navigation.less

```diff
@@ -1 +1,28 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+nav{
```

Comment:
One minor difference between your page and the design files is that your bottom dashed border extends to the width of the container, but the border extends to the _whole width_ of the page in the design files. Try tweaking this to make it better match!